rwengine/src/ai/CharacterController.cpp

```diff
@@ -215,9 +215,8 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
         if (character->getCurrentCycle() == cycle_open) {
             if (character->animator->isCompleted(AnimIndexAction)) {
                 tryToEnter = true;
-            } else if (entryDoor &&
-                       character->animator->getAnimationTime(AnimIndexAction) >=
-                           0.5f) {
+            } else if (entryDoor && character->animator->getAnimationTime(
+                                        AnimIndexAction) >= 0.5f) {
```

Comment:
This, I do not like. :(

Reply:
Yeah. It can be clearer and nicer to look at if the condition is extracted into a local const bool.

Reply:
Still not fixed

Reply:
Maybe `BreakBeforeBinaryOperators` could help.